RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -118,28 +189,31 @@ public bool Generate(ProcessedArgs args)
                 createdJarFilePath = BuildPlugin(definition, args.OutputDirectory);
             }
 
-            LogSummary(createdJarFilePath, generatedSqaleFile, licenseAcceptancePackages);
+            LogSummary(createdJarFilePath, generatedSqaleFile);
 
             return createdJarFilePath != null;
         }
 
-        private void LogSummary(string createdJarFilePath, string generatedSqaleFile, IEnumerable<IPackage> licenseAcceptancePackages)
+        private void LogSummary(string createdJarFilePath, string generatedSqaleFile)
         {
             if (generatedSqaleFile != null)
             {
-                // Log a messsage about the generated sqale file at the end of the process: if we
+                // Log a message about the generated sqale file at the end of the process: if we
```

Comment:
I don't think it's worth writing code to special-case out the dependencies. The code is unchanged, which is to say that one message per plugin generated is output into the log, which might cause it to be lost in the noise.